LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2020-2021 PostHog Inc.
```

Comment:
Given it's the 6th December do we go the whole hog (pun intended) and set this to 2023?

Reply:
Already got that lined up :D https://github.com/PostHog/posthog/pull/13143